Source/Shaders/PolylineCommon.glsl

```diff
@@ -32,30 +32,48 @@ void clipLineSegmentToNearPlane(
             clipped = true;
         }
     }
-    
+
     positionWC = czm_eyeToWindowCoordinates(vec4(p0, 1.0));
 }
 
-vec4 getPolylineWindowCoordinates(vec4 position, vec4 previous, vec4 next, float expandDirection, float width, bool usePrevious) {
+vec4 getPolylineWindowCoordinates(vec4 position, vec4 previous, vec4 next, float expandDirection, float width, bool usePrevious, out float angle) {
     vec4 endPointWC, p0, p1;
     bool culledByNearPlane, clipped;
-    
+
     vec4 positionEC = czm_modelViewRelativeToEye * position;
     vec4 prevEC = czm_modelViewRelativeToEye * previous;
     vec4 nextEC = czm_modelViewRelativeToEye * next;
-    
+
+    // Compute the window coordinates of the points.
+    vec4 positionWindow = czm_eyeToWindowCoordinates(positionEC);
```

Comment:
Can you `#ifdef` this out so polylines that don't use the angle don't compute it? For example, surround this code in `#ifdef POLYLINE_DASH` and in `PolylineDashMaterial.glsl` add `#define POLYLINE_DASH`.